contract-ui/types/types.ts

```diff
@@ -16,3 +16,8 @@ export type EnhancedRoute<T = any> = {
   isEnabled?: ExtensionDetectedState;
   component: ComponentType<T>;
 };
+
+export interface BasicContract {
+  chainId: number;
+  address: string;
+}
```

Comment:
seems indeed like a ver *basic* interface lol

Reply:
also prefer `type` unless you need to be able to `extend` it (which u don't I believe)

Reply:
very basic much wow